Assets/EngyneCreations/Trello/Scripts/TrelloBoardSettings.cs

```diff
@@ -0,0 +1,80 @@
+using System;
+using System.IO;
+using System.Linq;
+using UnityEngine;
+
+#if UNITY_EDITOR
+using UnityEditor;
+#endif
+
+namespace Trello
+{
+	public class TrelloBoardSettings : ScriptableObject
+	{
+		[Header("API Key and Token needed for Interacting with Trello")]
+		public string APIKey;
+		public string APIToken;
+
+		private static TrelloBoardSettings _instance;
+
+		public static TrelloBoardSettings Instance
+		{
+			get
+			{
+				LoadSettings();
```

Comment:
Should you LoadSettings everytime? 
or return `_instance` before Loading settings?

Reply:
So when reading on How `MixPanel` I realised that technically we should allow users change API tokens and keys whenever they want. So if I don't check `LoadSettings()` every time, if a users changes which Trello Workspace they want to use, the TrelloApi won't be able to update.

So loading each time is a safer way of doing I think

Reply:
So there's no reason to have `_instance`, and just return `Instance`?

Reply:
Yup sounds good. Was just copying and pasting the template of `MixPanel` but yeah can make the adjustments